cms/static/sass/elements/_footer.scss

```diff
@@ -58,5 +62,42 @@
         }
       }
     }
+
+    .footer-content-secondary {
+      @include clearfix();
+      margin-top: $baseline;
+    }
+
+    .footer-about-copyright, .footer-about-openedx {
+      display: inline-block;
+      vertical-align: middle;
+    }
+
+    // platform trademarks
+    .footer-about-copyright {
+      width: flex-grid(4, 12);
+      @include float(left);
+      margin-right: flex-gutter(2);
```

Comment:
Anytime the left and right margin are done independently, they should use the @include method which will flip the property, so this should be: 

```
@include margin-right(flex-gutter(2));
```

BUT I don't know how well it works with the flex-gutter. If it doesn't work, let me know and we can use a slightly more verbose method here.